base_product_mass_addition/models/product_product.py

```diff
@@ -44,3 +78,8 @@ def button_quick_open_product(self):
             "res_id": self.id,
             "target": "current",
         }
+
+    def check_access_rights(self, operation, raise_exception=True):
```

Comment:
Incorrect, by doing this you introduce a security issue as any body can write on a product by just passing an params in the context.

We only need to all a write on the field "quick_uom_id" and "qty_to_process".

If it's too complex, just remove this feature and we will do it latter in a specific PR

Reply:
OK, removing for later.

I have an idea about this (not sure, to test):
- check_access_rights: keep this code, but add:
- check_field_access_rights: call super(), but also verify the fields are either quick_uom_id and qty_to_process. Then if there is the context key, call again self.env['ir.model.access'].check() (which is what check_access_rights does) 

This means we get:
=> check_access_rights that's more permissive so that opens the permission for the user to write on the field, on the interface
=> check_field_access_rights that checks which fields we are writing
=> simulate another check_access_rights during check_field_access_rights so that if there is some inconsistency we get an AccessError on write()